Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -19,7 +62,23 @@ public static void Sort(int[] array)
         /// <param name="array">Input array in a random order.</param>
         public static void Reverse(int[] array)
         {
-            // ToDo: implement.
+            if (array == null || array.Length == 0)
```

Comment:
This condition could be a function to save the trouble of writing it over and over again.

Reply:
personal opinion for me is that this way its more readable and an extra method does not shorten the code, maybe even makes it longer ?
if (CheckForNullORZeroLength(array))
            {
                return;
            }

private static bool CheckForNullORZeroLength(int[] array)
        {
            if (array == null || array.Length == 0)
            {
                return true;
            }
            return false;
        }

Reply:
Edge case, it is good enough.